source/dofs/dof_tools_constraints.cc

```diff
@@ -1083,6 +1085,77 @@ namespace DoFTools
         }
     }
 
+    namespace internal
+    {
+      /**
+       * Try to find a finite element inside FECollection of @p dof_handler
+       * which is dominating those in @fes .
+       */
+      template <int dim, int spacedim>
+      unsigned int
+      find_dominating_fe (const dealii::hp::DoFHandler<dim,spacedim> &dof_handler,
+                          const std::set<unsigned int> fes)
+      {
+        unsigned int fe = numbers::invalid_unsigned_int;
+        unsigned int n_dofs = 0;
+        for (unsigned int cur_fe = 0; cur_fe < dof_handler.get_fe().size(); cur_fe++)
+          if (fes.find(cur_fe) == fes.end())
```

Comment:
I agree in general, but this function is called when `neither_element_dominates`. So I know that I shall skip the provided two elements.

Perhaps it would be better to rename it and/or add a note to the description.